e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js

```diff
@@ -96,7 +96,7 @@ describe('Image Link Preview', () => {
         cy.findAllByLabelText('file thumbnail').should('be.visible').and('have.length', 4);
     });
 
-    it('MM-T332 Image link preview - Bitly links for images and YouTube', () => {
+    it('MM-T332 Image link preview - Bitly links for images and YouTube -- KNOWN ISSUE: MM-40448', () => {
```

Comment:
I'm fine with this approach as we initially discussed offline. Please document this new process at https://developers.mattermost.com/contribute/webapp/e2e/guide-for-writing/

Reply:
@saturninoabril Created a doc PR https://github.com/mattermost/mattermost-developer-documentation/pull/1005

Reply:
Thanks!